crud.py

```diff
@@ -91,17 +122,19 @@ async def delete_market_product(product_id: str) -> None:
 async def create_market_zone(user, data: createZones) -> Zones:
     zone_id = urlsafe_short_hash()
     await db.execute(
-        """
+        f"""
         INSERT INTO market.zones (
             id,
             "user",
             cost,
-            countries
+            countries,
+            currency,
+            stall
```

Comment:
- `zones` should not know about `stalls`

Reply:
The goal is to have zones per stall. If they are "global", it causes issues with the shipping cost unit (sat, EUR, USD, ...)